src/controller/command-handlers/authenticate.cpp

```diff
@@ -59,11 +59,16 @@ QByteArray createAuthenticationToken(const QSslCertificate& certificate,
         {"x5c", QJsonArray({QString(certificateDer.toBase64())})},
     });
 
+    const QString sub =
+        certificate.subjectInfo("SN").isEmpty() && certificate.subjectInfo("GN").isEmpty()
```

Comment:
Seems that this does not work for some reason. I tested on Windows. More details in Jira.